src/Resources/contao/widgets/TableLookupWizard.php

```diff
@@ -10,7 +10,10 @@
  * @license    http://opensource.org/licenses/lgpl-3.0.html LGPL
  */
 
-class TableLookupWizard extends Widget
+namespace Contao;
```

Comment:
It must be in our namespace!

Reply:
It's not the "Symfony" way - it's the proper software engineering way :-) If you re-use anybody's namespace you're doing it wrong. Always :-)

Reply:
Yes you're right ^^